fe/fe-core/src/test/java/com/starrocks/catalog/JDBCResourceTest.java

```diff
@@ -0,0 +1,60 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+package com.starrocks.catalog;
+
+import com.google.common.io.FileBackedOutputStream;
+import com.starrocks.analysis.AccessTestUtil;
+import com.starrocks.analysis.Analyzer;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.*;
+import java.util.HashMap;
+import java.util.Map;
+
+public class JDBCResourceTest {
+    private Analyzer analyzer;
+
+    @Before
+    public void setUp() {
+        analyzer = AccessTestUtil.fetchAdminAnalyzer(true);
+    }
+
+    @Test
+    public void testSerialization() throws Exception {
+        // 1. Write objects to file
+        File file = new File("./jdbcResource");
+        file.createNewFile();
+        DataOutputStream dos = new DataOutputStream(new FileOutputStream(file));
+
+        Map<String, String> configs = new HashMap<>();
+        configs.put("hosts", "host1,host2");
+        configs.put("user", "user");
+        configs.put("password", "password");
+        configs.put("driver", "driver");
+        configs.put("jdbc_type", "jdbc_type");
+        configs.put("socketTimeout", "1000");
+        JDBCResource jdbcResourceWrite = new JDBCResource("jdbc_resource_test", configs);
+        jdbcResourceWrite.write(dos);
```

Comment:
Could we leverage some serialization testing framework, which could inject data into the serializable field automatic? Cause test serialization is boring

Reply:
I think you could refer to HiveResourceTest.testSerialization